src/sagemaker/serve/model_server/multi_model_server/server.py

```diff
@@ -25,16 +28,64 @@ class InProcessMultiModelServer:
 
     def _start_serving(self):
         """Initializes the start of the server"""
-        return Exception("Not implemented")
+        background_tasks = set()
+        task = asyncio.create_task(main())
+        background_tasks.add(task)
+        task.add_done_callback(background_tasks.discard)
+
+        time.sleep(10)
 
     def _invoke_multi_model_server_serving(self, request: object, content_type: str, accept: str):
-        """Invokes the MMS server by sending POST request"""
-        return Exception("Not implemented")
+        """Placeholder docstring"""
+        background_tasks = set()
+        task = asyncio.create_task(self.generate_connect())
+        background_tasks.add(task)
+        task.add_done_callback(background_tasks.discard)
 
     def _multi_model_server_deep_ping(self, predictor: PredictorBase):
```

Comment:
Shouldn't this actually ping the server similar to https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/serve/model_server/tei/server.py#L88?

Reply:
Good suggestion, I will tweak the code